v0/destinations/moengage/data/MOENGAGEIdentifyAttributesConfig.json

```diff
@@ -0,0 +1,118 @@
+[
+    {
+        "destKey": "name",
+        "sourceKeys": ["context.traits.name","traits.name"],
```

Comment:
Traits at top level should get higher preference.

Reply:
done